src/olympia/files/utils.py

```diff
@@ -127,38 +133,6 @@ def get_simple_version(version_string):
     return re.sub('[<=>]', '', version_string)
 
 
-class JSONExtractor(object):
-    def __init__(self, path, data=''):
-        self.path = path
-
-        if not data:
-            with open(self.path) as fobj:
-                data = fobj.read()
-
-        lexer = JsLexer()
-
-        json_string = ''
-
-        # Run through the JSON and remove all comments, then try to read
-        # the manifest file.
-        # Note that Firefox and the WebExtension spec only allow for
-        # line comments (starting with `//`), not block comments (starting with
-        # `/*`). We strip out both in AMO because the linter will flag the
-        # block-level comments explicitly as an error (so the developer can
-        # change them to line-level comments).
-        #
-        # But block level comments are not allowed. We just flag them elsewhere
-        # (in the linter).
-        for name, token in lexer.lex(data):
-            if name not in ('blockcomment', 'linecomment'):
-                json_string += token
-
-        self.data = json.loads(unicodehelper.decode(json_string))
-
-    def get(self, key, default=None):
-        return self.data.get(key, default)
```

Comment:
We don't inherit from it anywhere else so I merged it with `ManifestJSONExtractor`.